mkdocs/docs/frontends/tensorflow.md

```diff
@@ -29,6 +29,56 @@ python -m pip install \
 
 ## Importing models
 
+### From SavedModel on TensorFlow Hub
+
+IREE supports importing and using SavedModels from
+[TensorFlow Hub](https://www.tensorflow.org/hub).
+
+#### Using the command-line tool
+
+IREE compilers transform a model into its final deployable format in many
+sequential steps. The first step is to import the model authored with Python
+in TensorFlow into a format (i.e., [MLIR](https://mlir.llvm.org/)) expected
+by main IREE compilers. This can be done via the `iree-tf-import` tool.
```

Comment:
Let's shuffle this paragraph around a bit. The general steps apply to SavedModels not downloaded from TF Hub _and_ to using the Python compiler APIs (which invoke the command line tool(s)).

Also included in my suggestion:

* "many sequential steps" ->"several sequential steps": rewording to be less daunting to users ;)
* "expected by main IREE compilers" -> "compatible with the generic IREE compilers": not sold on either language, but I want to hint more strongly that each framework importer acts in a similar way - converging on common formats

```suggestion
## Importing models

IREE compilers transform a model into its final deployable format in several
sequential steps. The first step for a TensorFlow model is to use either the
`iree-tf-import` command-line tool or IREE's Python APIs to import the model
into a format (i.e., [MLIR](https://mlir.llvm.org/)) compatible with the generic
IREE compilers.

### From SavedModel on TensorFlow Hub

IREE supports importing and using SavedModels from
[TensorFlow Hub](https://www.tensorflow.org/hub).

#### Using the command-line tool
```

Reply:
Cool. I'll just blatantly copy your words here. :)